cluster-autoscaler/processors/provreq/injector.go

```diff
@@ -108,3 +117,7 @@ func NewProvisioningRequestPodsInjector(kubeConfig *rest.Config) (pods.PodListPr
 	}
 	return &ProvisioningRequestPodsInjector{client: client, clock: clock.RealClock{}}, nil
 }
+
+func key(pr *provreqwrapper.ProvisioningRequest) string {
+	return strings.Join([]string{pr.Namespace, pr.Name}, " ")
```

Comment:
Maybe we could use UID here? In case user notices the problem and recreates the request

Reply:
Done